TelemetrySystem/src/main/java/tddmicroexercises/telemetrysystem/TelemetryDiagnosticControls.java

```diff
@@ -0,0 +1,50 @@
+package tddmicroexercises.telemetrysystem;
+public class TelemetryDiagnosticControls implements TelemetryDiagnosticService
+{
+    private final String DiagnosticChannelConnectionString = "*111#";
+    private final TelemetryClient telemetryClient;
+    private final TelemetryClientConnection telemetryClientConnection;
+    private String diagnosticInfo = "";

```

Comment:
the type of telemetryClient should be the interface type that is TelemetryClientService, same for telemetryClientConnection.... to ensure dependency inversion principle